apps/wfo-ui/next.config.js

```diff
@@ -7,4 +7,13 @@ module.exports = {
         defaultLocale: 'en-US',
     },
     transpilePackages: ['@orchestrator-ui/orchestrator-ui-components'],
+    async redirects() {
```

Comment:
If you implement my suggested solution, this config is not needed anymore

Reply:
Fixed the redirect with middleware: no need for a dummy page.